provisioning/transport/amqp/src/AmqpLinkFactory.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Amqp;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.Transport.Amqp
+{
+    class AmqpLinkFactory : ILinkFactory
+    {
+        public IAsyncResult BeginOpenLink(AmqpLink link, TimeSpan timeout, AsyncCallback callback, object state)
```

Comment:
Not used anywhere - I think we can delete.

Reply:
interface member

---
In reply to: [146718068](https://github.com/Azure/azure-iot-sdk-csharp/pull/204#discussion_r146718068) [](ancestors = 146718068)